mucgpt-frontend/src/pages/layout/Layout.module.css

```
@@ -148,22 +148,82 @@
     width: 100%;
 }
 
+.mobileUtilityRow > div {
+    width: 100%;
+}
+
+.mobileUtilityRow > div:not([class*="container"]) {
+    width: 100%;
+}
+
 .mobileUtilityRow :global(.fui-Button),
```

Stylelint (17.12.0)

[error] Unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)